ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -44,6 +48,22 @@ public PatientVisitsDashboardPage(Page parent, WebElement waitForStaleness) {
     public String getPageUrl() {
         return "coreapps/patientdashboard/patientDashboard.page";
     }
+    
+    public void goToEditVitalsPage() {
+        clickOn(EDIT_VITALS_ICON);
+    }
+    
+    public void goToViewVitalsPage() {
+    	clickOn(VIEW_VITALS_ICON);
+    }
+    
+    public void deleteVitals() {
+    	clickOn(DELETE_VITALS_ICON);
+    }
```

Comment:
The method deleteVitals() sounds to do all the transactions of a delete method(), so why again creating another method confirmDeleteVitals()?

Reply:
@kdaud that method deleteVitals() does not fully complete the action, it just brings up the modal for deleting then you have to call the confirmDeleteVitals() method to complete the action of deleting the vitals or encounter.

Reply:
The method deleteVitals() sounds to do all the transactions of a delete method(), there are two ideas to think of;

1. Renaming the method from `deleteVitals() `to some other name like `clickOnDeleteVitalsIcon() `
 OR
2. Retaining the method name and refactor `confirmDeleteVitals()` into `deleteVitals()`. With this option, we can have something like;
```js
public void deleteVitals() {
        clickOn(DELETE_VITALS_ICON);
        waitForElement(COMFIRM_DELETE_BUTTON);
        clickOn(COMFIRM_DELETE_BUTTON);
    }
```

Reply:
@kdaud  I think option 2 looks good to me. I think let me implement the second one.

Reply:
Wait a minute @kdaud, if I gave it a second thought, let me implement the first option just to make sure that these two methods are not coupled.  What do you think @kdaud .

Reply:
> What do you think @kdaud .

I would go with your first thought !